junit-reporting-handlebars/src/main/java/com/github/bogdanlivadariu/reporting/junit/builder/JUnitReportBuilder.java

```diff
@@ -3,6 +3,7 @@
 import java.io.File;
 import java.io.FileNotFoundException;
 import java.io.IOException;
+import java.lang.ClassCastException;
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Avoid importing anything from the package java.lang](https://www.codacy.com/app/bogdan-livadariu/bootstraped-multi-test-results-report/file/2695613442/issues/source?bid=3293946&fileBranchId=3336457#l6)